hs/src/Reach/Eval/Core.hs

```diff
@@ -4264,9 +4265,11 @@ evalExpr e = case e of
     locAtf (srcloc_jsa "id ref" a) $
       evalId "expression" x
   JSDecimal a ns -> do
-    when ('e' `elem` ns || 'E' `elem` ns) $
-      expect_ $ Err_Eval_IllegalJS e
-    case splitOn "." ns of
+    let ns' = case ('e' `elem` ns || 'E' `elem` ns) of
+            True -> do
+              showFFloat Nothing (read ns :: Double) ""
```

Comment:
Cute. You need some tests though and to review the CI logs

Reply:
And to update the docs... it says we don't support it

And make sure you update the changelog to say that we do now too